components/Feature/VulnerabilitiesGrid/grid.json

```diff
@@ -8,7 +8,13 @@
       },
       {
         "label": "Other",
-        "textinputs": [{ "type": "other", "label": "Other assets" }]
+        "arialabel": "Other assets",
+        "textinputs": [
+          {
+            "type": "other",
+            "label": "Other assets"
```

Comment:
As it's hardcoded anyway, would it make sense to have it as a "Other financial stability assets" ? Cause there is other in every section (unless it's clear in other ways)

Reply:
The screen reader tells them they are in the financial stability section